src/components/tabs.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { FC, ReactNode, useState } from "react"
+import classnames from "classnames"
+
+interface Tab {
+  name: string
```

Comment:
I'm not so good with names. But we could find another name for this prop, maybe `label`, `text`. 
`name` is too abstract in my point of view. But up to you.

Reply:
Makes sense, I went with `title` seems to be more specific and encapsulates what the prop actually does. Thanks for the push.